image.h

```diff
@@ -35,3 +35,21 @@ class Image {
 	const uint16_t m_columns;
 	uint8_t* m_pixels; // max 1k x 1k image
 };
+
+class RawImage : public Image
```

Comment:
Nice approach to derive these types from `Image` itself. There could be functions that don't care whether they take raw or brightened images - they can use `Image` 👍